typings/commander-tests.ts

```diff
@@ -112,6 +122,8 @@ program
 program
     .command("name1", "description")
     .command("name2", "description", { isDefault:true })
+    .command("name3", "description").action(syncCall)
```

Comment:
Commands with action handlers should not supply description. Which we can not enforce with typing, sadly! I like adding it here as name3, name4 though. But name 3 and name 4 need to be added to program alone. e.g.
```
program
  .command("name3)
  .action(syncCall);
```

Reply:
thanks! :) Fixed